src/plugins/apresTimeline/plugin.js

```diff
@@ -0,0 +1,127 @@
+import ActivityViewProvider from './activityViewProvider';
+import TimelineViewProvider from './timelineViewProvider';
+import ChroniclesViewProvider from './chroniclesViewProvider';
+
+
+export default function () {
+    return function install(openmct) {
+        openmct.types.addType(
+            'apres.timeline.type',
+            {
+                name: 'Apres Timeline',
+                cssClass: 'icon-timeline',
+                creatable: true,
+                initialize: function (domainObject) {
+                    domainObject.composition = [];
+                }
+            }
+        );
+        openmct.types.addType(
+            'apres.timeZone.type',
+            {
+                name: 'Apres Time Zone',
+                cssClass: 'icon-hourglass',
+                creatable: true
+            }
+        );
+        openmct.types.addType(
+            'apres.chronicle.type',
+            {
+                name: 'APRES State Chronicle',
+                cssClass: 'icon-arrow-left',
+                creatable: true,
+                
```

Comment:
We need an initialize function like in the activity type, which should have domainObject.configuration, which will include the stateName, startTime and endTime with initial values

Reply:
With the icon issue, its showing an unknown type in the second picture. Delete these via right click, and create a new chronicle, and see if that works